R/param-estimates.R

```diff
@@ -8,6 +8,12 @@
 #' Returns a tibble containing parameter estimates from a `bbi_{.model_type}_summary` object.
 #' Details about the tibble that is returned are in the return value section below.
 #'
+#' @details
+#' Note that **Bayesian methods are not yet supported** by this function. Creating a parameter table
+#' like this for Bayesian estimation methods requires a different approach, which has not yet been
+#' implemented. If the final estimation method of the input model is Bayesian, a `not implemented`
```

Comment:
This is a place to consider [custom conditions](https://adv-r.hadley.nz/conditions.html#custom-conditions).

Reply:
Yes, this is a good point and something I've considered in the past. Devin pointed me towards reading up on `rlang::abort()` awhile ago with this in mind.

I would love to address this at some point, but given our current time constraints, I think I'm gonna punt. Good to keep reminding ourselves of this though. I love errors with classes. (I miss thee Python...)